TheMovieDB/MovieDetailViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  MovieDetailViewController.swift
+//  TheMovieDB
+//
+//  Created by Juliana Lima on 30/10/16.
+//  Copyright © 2016 Juliana Lacerda. All rights reserved.
+//
+
+import UIKit
+import AlamofireImage
+
+class MovieDetailViewController: UIViewController {
+
+    @IBOutlet weak var moviePoster: UIImageView!
+    @IBOutlet weak var movieTitle: UILabel!
+    @IBOutlet weak var movieRelease: UILabel!
+    @IBOutlet weak var movieGenres: UILabel!
+    @IBOutlet weak var movieOverview: UILabel!
+    
+    public var movieData: Movie!
```

Comment:
Declaring a public property as implicitly unwrapped optional is not safe. You have no guarantee that it has a value and this could cause crashes.